Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/contest-card/ContestCard.tsx

```diff
@@ -40,13 +41,17 @@ const ContestCard = ({ contest }: IContestCardProps) => {
 
     const renderCountdown = useCallback(
         () => {
-            if (canBePracticed && practiceEndTime == null) {
+            const endDate = canBeCompeted
+                ? endTime
+                : practiceEndTime;
+
+            if (canBePracticed && isNil(practiceEndTime) && isNil(endDate)) {
                 return <p>No practice end time.</p>;
             }
 
-            const endDate = canBeCompeted && !canBePracticed
-                ? endTime
-                : practiceEndTime;
+            if ((!canBePracticed && !canBeCompeted) || isNil(endDate)) {
+                return <p>No available remaining time.</p>;
```

Comment:
This message kind of looks like something is wrong. I would not show any message, just return `null` (if this does not break the component/appearance).